templates/javascript.js.twig

```diff
@@ -356,3 +356,30 @@ $(document).ready(function() {
         {% endfor %}
     });
 {% endmacro %}
+
+{% macro dialog(selector) %}
+$(document).ready(function() {
+    var element = $("{{ selector }}");
+
+    element.css("visibility", "visible");
+    element.dialog({
+        height: "auto",
+        width: "90%",
+        modal: true,
+        minHeight: 500,
+        position: { my: "center", at: "top" },
+        buttons: {
+            "{{ 'referent.button.accept'|trans }}": function () {
+                $.ajax({
+                    url: "{{ path('app_user_set_accept_chart')|e('js') }}"
+                });
+
+                $(this).dialog("destroy");
+            }
+        },
+        beforeClose: function (event, ui) {
+            window.location.replace("{{ path('homepage') }}");
```

Comment:
why this redirection?

Reply:
Asked by PO, user has to accept or he is redirected to the homepage

Reply:
can you make this macro more generic to reuse it easly ? or rename it ?